<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomCollageShape</class>
 <widget class="QDialog" name="CustomCollageShape">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Custom Collage Shape</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnBrush">
       <property name="text">
        <string>Brush</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEraser">
       <property name="text">
        <string>Eraser</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Brush size</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QSlider" name="sliderBrushSize">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="graduation_100">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QLabel" name="graduation_80">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>80</string>
        </property>
       </widget>
       <widget class="QLabel" name="graduation_60">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>87</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>60</string>
        </property>
       </widget>
       <widget class="QLabel" name="graduation_40">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>115</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>40</string>
        </property>
       </widget>
       <widget class="QLabel" name="graduation_20">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>142</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>20</string>
        </property>
       </widget>
       <widget class="QLabel" name="graduation_0">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>172</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="DrawingArea" name="drawingArea" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="presetLabel">
       <property name="text">
        <string>By default :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxShapeImage">
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>File :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingArea</class>
   <extends>QWidget</extends>
   <header>drawingarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomCollageShape</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomCollageShape</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBrush</sender>
   <signal>clicked()</signal>
   <receiver>btnEraser</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEraser</sender>
   <signal>clicked()</signal>
   <receiver>btnBrush</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
